pkg/backend/display/progress.go

```diff
@@ -585,12 +585,20 @@ func (display *ProgressDisplay) printPolicies() bool {
 	for name, version := range display.summaryEventPayload.PolicyPacks {
 		var summary policyPackSummary
 		baseName, path := engine.GetLocalPolicyPackInfoFromEventName(name)
+		var key string
 		if baseName != "" {
-			summary.IsLocal = true
-			summary.LocalPath = path
-			name = baseName
+			key = fmt.Sprintf("%s@v%s", baseName, version)
+			if s, has := policyPackInfos[key]; has {
+				summary = s
+				summary.LocalPaths = append(summary.LocalPaths, path)
```

Comment:
Shouldn't this be setting IsLocal=true as well?

Reply:
In theory it should be set already, unless there's also a remote policy pack that has the same name, which I guess could be possible :thinking:

Reply:
The latest commit now addresses that.  It's a bit of a weird situation, and I don't really expect it to happen, but If it does it now shows something like

`<name>@<version> (local: <path>;...) (cloud)`. Not sure if there's a better way to address this.